binnacle/lib/CompassWidget.dart

```diff
@@ -1,31 +1,36 @@
 import 'package:flutter/material.dart';
```

Comment:
Think we should change the name of this class to BinnacleWidget, along with changing CompassUI to BinnacleUI